templates/lib/before_deploy.sh

```diff
@@ -6,7 +6,8 @@ set -ex
 
 main() {
   local src=$(pwd) \
-    stage=
+        stage= \
+        linking_args=
```

Comment:
could we get 2 lines indentation here?

Reply:
I've updated emacs to use 2 spaces as identation, but it keeps wanting to align the variables instead of indenting it.
Should they be indented instead of aligned then?

Reply:
Yeah, I don't align any of my code; it's usually harder to have people collaborate on (unless it's enforced by something like `gofmt` / `rustfmt`).